js-exercises/rot-13/rot13.js

```diff
@@ -0,0 +1,19 @@
+function rot13(cipherString) {
+  let result = '';
+  if (typeof (cipherString) === 'string') {
+    let splitString = cipherString.trim().split(' ');
+    let decipherString = [];
+    for (let word of splitString) {
+      const temp = word.split('')
```

Comment:
always declare any constant or variable before starting the loop cuz it will initialize the var/const for one time and change the value/content of const i.e array or object.

Reply:
Okay I will keep this in mind